### ISSUE TITLE
Registered URLs for handlers

### ISSUE THREAD

**Issue by stefanberger** [keylime/keylime#644]
This is from here: https://github.com/keylime/keylime/blob/master/keylime/cloud_verifier_tornado.py#L815

```
    app = tornado.web.Application([
        (r"/(?:v[0-9]/)?agents/.*", AgentsHandler),
        (r"/(?:v[0-9]/)?allowlists/.*", AllowlistHandler),
        (r".*", MainHandler),
    ])
```

Am I reading this right that these handlers are reachable under `/v2/allowlist` and `v2/agents` and also `v3/allowlist` and so on?

**Comment by kaifeng** [comment]
It is, currently some handlers check version but some are not.

**Comment by stefanberger** [comment]
I thought `/v2/*`  should be reserved for a revised API after a v1. But now these registrations are already covering v0 ... v9.

**Comment by stefanberger** [comment]
API versioning is now implemented via #701
